README.md

```diff
@@ -0,0 +1,33 @@
+# ViaRE's Reference
+
+This is the place in where we share the list of references we use to keep developing
+[ViaRE](https://github.com/daque-dev/viare).
+
+## Content
+- [The proccess](#the-process) (how to add more references to this repo)
+
+## The proccess
+
+Every reference we find must go through this proccess:
+
+- **The Finding**: We find a book, website, article, magazine, etc. that we consider
+may be useful to keep generating ideas, getting context, or keeping us interested.
+When we skim the table of contents or something similar, we should take note of what
+the book could give to ViaRe, and then the book makes it to the `finding list`.
+
+- **Note-taking**: When any member of [Daque](https://github.com/orgs/daque-dev/people)
+starts to read an item inside `finding list`, he/she must start to take notes on that
```

Comment:
`finding list` could be either a file `FINDING.md` or a branch `finding`
I think we must agree on something.

Reply:
I see more feasible a file. I strongly support that approach.

Reply:
I don't understand how it would work being a branch.

Reply:
I can only think very abstractly about it, so we better use a file. We all agree?

Reply:
Yei.